android/examples/run-local-test/features/steps/steps.py

```diff
@@ -34,7 +12,7 @@ def step_impl(context):
     WebDriverWait(context.browser, 30).until(
         EC.presence_of_element_located((MobileBy.CLASS_NAME, "android.widget.TextView"))
     )
-        
+    
 @then(u'page contains "{regex}"')
 def step_impl(context, regex):
     test_element = None
```

Comment:
I see difference between code that is specified in docs: https://www.browserstack.com/docs/app-automate/appium/getting-started/python/behave/local-testing and what is in our code samples: Are they supposed to be same?

Reply:
I had taken the test implementation from what is currently in the master. Will update code snippets in the documentation.